.github/workflows/build_and_test.yml

```diff
@@ -417,10 +417,6 @@ jobs:
     - name: Free up disk space
       shell: 'script -q -e -c "bash {0}"'
       run: |
-        if [[ "$MODULES_TO_TEST" != *"pyspark-ml"* ]] && [[ "$BRANCH" != "branch-3.5" ]]; then
```

Comment:
here doesn't make much sense any more, since we install those libraries for each python version